JordanPhillips_ImJKP/hw_2/array_exercises.rb

```diff
@@ -0,0 +1,52 @@
+# Find the length of strings in an array
+
+def length_finder(input_array)
+  lengths=[]
+  input_array.each do |item| 
+    if item.is_a? String
+      lengths << item.length
+    end
+  end
+  lengths
```

Comment:
Yay for quality code! Very cool that you specified the data type should be a string.